public/js/loginValidation.js

```diff
@@ -13,28 +13,31 @@ function isValidEmail(elementID) {
     }
 }
 
+
+function postLoginDetails(email, password){
+    fetch("/api/login",
+        {
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            },
+            method: "POST",
+            body: JSON.stringify({
+                "userEmail": email,
+                "password": password
+            })
+        })
+        .then(function(data){ return data.json() })
+        .then(function (value) {console.log(value) })
+}
+
+
 function validateToPostInputs() {
```

Comment:
Sorry to be picky, but this may need to be more specific as we will have to validate & post other inputs later on - maybe validateLoginInput?